aspnet/conceptual-overview/aspnet.rst

```diff
@@ -104,8 +124,23 @@ ASP.NET Core uses a new configuration model for handling of simple name-value pa
 
 See :doc:`/fundamentals/configuration` for more details on the new configuration system and :doc:`/fundamentals/environments` for details on how to work with environments in ASP.NET Core.
 
+Build web UI and web APIs using MVC
```

Comment:
While I don't know MSFT's policy, the AP likes web and internet lowercase these days. http://www.poynter.org/2016/ap-style-change-alert-dont-capitalize-internet-and-web-any-more/404664/